experiments/minimal_net.py

```diff
@@ -6,11 +6,14 @@
 
 sys = system.System()
 
+# create disk images
+distro_disk_image = system.DistroDiskImage(sys, "base")
```

Comment:
For now this seems fine, but may be be better to have these "distro" instances pre-created in a module somewhere in the future?